Resources/views/Form/formatter.html.twig

```diff
@@ -49,6 +49,14 @@
                         appendClass = 'html';
                         break;
                     case 'richhtml':
+                        {%- for name, plugin in ckeditor_plugins if ckeditor_plugins is defined -%}
+                            CKEDITOR.plugins.addExternal('{{ name }}', '{{ plugin.path }}',
```

Comment:
@rande :  oh ok. After discussing with @roukmoute , it appears that modifying the current instance is totally possible. @roukmoute is going to try that soo.